tests/MoverGroupper.test.tsx

```diff
@@ -969,4 +969,115 @@ describe("MoverGroupper", () => {
                 );
             });
     });
+
+    it("should handle tabbing in groupper/mover/groupper when the inner groupper container is delegated and not focusable", async () => {
```

Comment:
This test is massive, what if you create a separate suite of delegated groupper and split this test up?

```tsx
describe('delegated groupper', () => {
  it('tabs into delegated groupper);
  it('shift+tabs into delegated groupper);
  it('mover focuses on first element of delegated groupper);
})
```

Reply:
This test has average size for all tests in this file as they test various mover/groupper combinations.
There are more related tests in Groupper.test.tsx (look for `delegated` word in the test name).